build.sh

```diff
@@ -10,9 +10,20 @@ fi
 BUILD_TARGET="$1"
 VERSION=$(cat ./VERSION)
 
+BUILDER="docker build"
+if ! ${BUILDER%% *} -v >/dev/null 2>&1; then
+    BUILDER="buildah bud"
+    if ! ${BUILDER%% *} -v >/dev/null 2>&1; then
```

Comment:
Rather than nesting your `if` statements, just use an `if`, `elif`, `else` block to simplify this process.  
```suggestion
elif ! buildah -v >/dev/null 2>&1; then
```